packages/cli/src/databases/migrations/sqlite/1660062385367-CreateCredentialsEditorRole.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import * as config from '../../../../config';
+import { logMigrationEnd, logMigrationStart } from '../../utils/migrationHelpers';
+
+export class CreateCredentialsEditorRole1660062385367 implements MigrationInterface {
```

Comment:
Considering the current scope does not allow actually editing the credential when it's shared, should we rename this to `CreateCredentialsUserRole` instead?